task/sql-tasks.js

```diff
@@ -177,7 +272,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT 
+            COUNT(ProductID) AS "TotalOfCurrentProducts",
+            SUM(Discontinued) AS "TotalOfDiscontinuedProducts"

```

Comment:
use count, it shows intent better
even better just write 2 subqueries and a select without from-clause

eg

```
select 
 (select count(*) ....) as 'T'
 (select count(*) ....) as 'TT'
```